packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -548,7 +553,7 @@ class HotRunner extends ResidentRunner {
       benchmarkData['hotReloadMillisecondsToFrame'] =
           reloadTimer.elapsed.inMilliseconds;
     }
-    if (shouldReportReloadTime)
+    if (!reassembleTimeout && shouldReportReloadTime)
```

Comment:
Not sure if reload time should be reported if there are other sorts of errors, too. Check reassembleAndScheduleErrors here too?